portal/flatpak-portal.c

```diff
@@ -1600,6 +1600,7 @@ typedef struct {
   int n_ops;
```

Comment:
Should we document these guarantees in the interface docs ?

Reply:
Probably a good idea.